packages/devtools-reps/src/reps/grip-map.js

```diff
@@ -49,6 +49,9 @@ function GripMap(props) {
   }
 
   const propsArray = safeEntriesIterator(props, object, maxLengthMap.get(mode));
+  if (mode == MODE.EXPANDED) {
```

Comment:
I think we could add a condition on the if statement of line 48, since we return the same thing

Reply:
Ops! I'll fix it now.